scripts/zones/Metalworks/npcs/Iron_Eater.lua

```diff
@@ -45,9 +45,9 @@ function onTrigger(player,npc)
 		player:startEvent(0x02cc);
 	elseif(currentMission == THE_CHAINS_THAT_BIND_US and missionStatus == 0) then
 	    player:startEvent(0x02ff); -- First cutscene of mission
-	elseif (currentMission == THE_CHAINS_THAT_BIND_US) and (missionStatus == 2) then
+	elseif (currentMission == THE_CHAINS_THAT_BIND_US and missionStatus == 2) then
 		player:showText(npc, 8596); -- Dialogue after first cutscene

```

Comment:
While its def something I'd change too (original line was bad styling) it wouldn't actually cause any functional difference, so something else had to be causing the problem.